vllm/entrypoints/openai/cli_args.py

```diff
@@ -54,11 +54,15 @@ def make_arg_parser():
                         help="If provided, the server will require this key "
                         "to be presented in the header.")
     parser.add_argument("--served-model-name",
+                        nargs="+",
                         type=str,
                         default=None,
-                        help="The model name used in the API. If not "
-                        "specified, the model name will be the same as "
-                        "the huggingface name.")
+                        help="The model name(s) used in the API. If multiple "
+                        "names are provided, the server will respond to any "
+                        "of the provided names. The model name in the model "
+                        "field of a response will be the first name in this "
+                        "list. If not specified, the model name will be the "
+                        "same as the huggingface name.")
```

Comment:
Not technically part of the PR changes, but related. `--model-name` might be an explicit path to a local model directory containing a huggingface model. In this case, the current logic will expect that path as the model name in API requests (so the `--served-model-name` should probably always be used in these cases).
```suggestion
                        "same as the --model-name argument.")
```

Reply:
I like the suggestion, but `--model-name` is incorrect, it's `--model`:

```suggestion
                        "same as the `--model` argument.")
```